res/controllers/Novation-Launchpad MK2-scripts.js

```diff
@@ -17812,7 +17811,7 @@ var console$1 = {
 };
 
 /* eslint-disable no-global-assign, no-native-reassign */
-console = console$1;
+var console = console$1;
```

Comment:
Thanks for this patch. I included it in [v2.2.1](https://github.com/dszakallas/mixxx-launchpad/releases/tag/v2.2.1)